tests/test_s3.py

```diff
@@ -3678,6 +3679,27 @@ def test_delete_bucket_encryption(self):
         with self.assertRaises(Exception):
             client.get_bucket_encryption(Bucket=bname)
 
+    @mock.patch('c7n.actions.invoke.assumed_session')
+    def test_s3_invoke_lambda_assume_role_action(self, mock_assumed_session):
```

Comment:
there aren't any actual assertions in this test, we need to assert that the assumed session is actually being used